src/engine/objects/mod.rs

```diff
@@ -547,6 +547,9 @@ where
             }
             PropertyKind::Scalar => {
                 resolver.resolve_scalar_field(info, field_name, &self.fields, executor)
+            },
+            PropertyKind::ScalarComp => {
```

Comment:
Is this a possibility? Or if it is, shouldn't it be an error?  Wouldn't ScalarComp fields only ever show up inside `Input` structs? If one ever appeared inside a `Node` struct with a resolver being called on it, wouldn't it be an error, like the `PropertyKind::Union` variant below?

Reply:
Yes - good catch.